### PR TITLE
OrderHistory Component

### LINKED ISSUE #2586: [Rx] OrderHistory component
Sortable list of recent orders, plus header, and select menu of sorting criteria.

Should look like a table on desktop. Rows should stack into a list on mobile.

### Prototypes 

- [Mobile](https://marvelapp.com/iaa9b9#14368789)
- [Desktop](https://marvelapp.com/1h10heg#14227011)

### PR DESCRIPTION
**Pull request by U-DON**
Closes #2586.

Created OrderHistory component, added it into the Detail page, and populated it with data from the API.

Excuse the wonky spacing and heading sizes.
Not sure if we're supposed to override those or if it's something Vanguard handles later.
## Relevant Mocks

Desktop: https://marvelapp.com/1h10heg#14227011
Mobile: https://marvelapp.com/iaa9b9#14368789
## Desktop Preview

screenshot from 2016-08-29 12 55 04
## Mobile (Small Screen) Preview

screenshot from 2016-08-29 12 55 24

### CODE REVIEW

**Comment by U-DON** [comment]
I still have questions about the sorting feature of OrderHistory (#2809), so haven't gotten to that yet. If there's work to be done there, I'm thinking of doing that on a separate branch.

File: src/js/rx/actions/prescriptions.js [hunk 1]
```diff
@@ -3,12 +3,17 @@ export function loadData(id) {
...
     // when API is able to retrieve any individual Rx.
     const testId = 1435525;
-    return dispatch => fetch(`/rx-api/prescriptions/${testId}`)
-      .then(res => res.json())
-      .then(
-        data => dispatch({ type: 'LOAD_PRESCRIPTION_SUCCESS', data }),
+    const rxUrl = `/rx-api/prescriptions/${testId}`;
+    const rxUrls = [rxUrl, `${rxUrl}/trackings`];
+    return dispatch => {
+      Promise.all(rxUrls.map(url => fetch(url).then(res => res.json()))).then(
```
Comment (alexose) [inline src/js/rx/actions/prescriptions.js]: This is awesome, but a little hard to follow.  Maybe break it into separate lines?

File: src/js/rx/components/OrderHistory.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  }
+
+  render() {
+    const rows = this.props.items.map(this.makeRow);
+
+    return (
+      <table className={this.props.className}>
+        <thead>
+          <tr>
+            <th>Order status</th>
```
Comment (alexose) [inline src/js/rx/components/OrderHistory.jsx]: The mock has an "Order date" column-- Is that still current?

**Comment by alexose** [comment]
A few notes above, but, this is looking great.

> Not sure if we're supposed to override those or if it's something Vanguard handles later.

I'd override them in the most reasonable way you can, then look for a way to make them generic later on.

File: src/js/rx/components/OrderHistory.jsx (hunk 2 above)
Comment (U-DON) [inline src/js/rx/components/OrderHistory.jsx]: There's currently no way to pull order date from the tracking history, so @ayaleloehr said to leave it off for now.

**Comment by U-DON** [comment]
Tweaked the styling to get the page looking better and closer to the mocks.
Note that the order history should be most recent first, but that change is happening on the API side.

## Desktop Preview
screenshot from 2016-08-29 19 02 11

## Mobile (Small Screen) Preview
screenshot from 2016-08-29 19 01 56
